common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -322,36 +323,39 @@ private ServiceException getServiceException(@NonNull final BrokerResult brokerR
      * @param records List of AccountRecord
      * @return Bundle
      */
-    public Bundle bundleFromAccountRecordList(@NonNull final List<AccountRecord> records) {
+    public Bundle bundleFromCacheRecordList(@NonNull final List<ICacheRecord> records) {
         final Bundle resultBundle = new Bundle();
-        ArrayList<String> accountRecordString = new ArrayList<>();
-        for (AccountRecord record : records) {
-            final String recordInGson = new Gson().toJson(record, AccountRecord.class);
-            accountRecordString.add(recordInGson);
+        ArrayList<String> cacheRecordString = new ArrayList<>();
+        if (records != null && !records.isEmpty()) {
+            for (ICacheRecord record : records) {
+                final String recordInGson = new Gson().toJson(record, CacheRecord.class);
+                cacheRecordString.add(recordInGson);
+            }
         }
 
-        resultBundle.putStringArrayList(BROKER_ACCOUNTS, accountRecordString);
+        resultBundle.putStringArrayList(BROKER_ACCOUNTS, cacheRecordString);
```

Comment:
Can we make this implementation match: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/520/files#diff-44f2260eafcc843207898d147d7285beR388 ?